apps/www/_blog/2022-08-17-supabase-soc2.mdx

```diff
@@ -0,0 +1,111 @@
+
+## Choosing a compliance monitoring tool
+
+There are a few tools that plug into your cloud providers, task management system, identity, and HR providers and make it easy to monitor, collect and submit evidence to auditors. We evaluated [Vanta](https://www.vanta.com/), [Drata](https://drata.com/), and [Secureframe](https://secureframe.com/). Our stack is pretty standard (AWS, GCP, GSuite, and Github). All the tools we evaluated were integrated with these systems.
+
+Some of them had overly broad permissions when integrating with our vendors (read/write access to Github for example) which made them a no-go. Some came with a lightweight agent that we could use for Mobile Device Management (MDM). Other than that, the products are largely the same.
+
+We chose Vanta, with Drata coming in a close second. Vanta supported more compliance standards at that time and is a fellow YC company. The gap between these tools is much closer now since all of them support the compliance standards we have our eyes on.
+
+## Choosing an auditor
+
+After setting up Vanta (nb: pestering devs to install a security agent on their laptop) and working through the checklist of tasks required for SOC2, we started looking around for auditors. Vanta was helpful here - they made introductions to a few auditors and we chose one who has previously worked with a lot of SaaS companies. Our auditors were also familiar with how Vanta worked which made the audit easier for us.
+
+The engagement started with a scoping call where we reviewed the systems which are in scope for the audit. We then set a date at which the audit takes place. The Type 1 audit checks if we are adhering to our stated policies at this point in time.
+
+## Off to the races
+
+Once the date for the audit was set, it was a mad scramble to get everything in order.
+
+Vanta has a lightweight read-only agent built on top of [osquery](https://github.com/osquery/osquery) to verify if all team members have important security features enabled: hard disk encryption, screen locks enabled, etc. It supports Mac, Windows, and Linux, although we had [some issues](https://github.com/system76/firmware-open/issues/268) with the agent running correctly in Linux. Getting hard disk encryption right is still difficult in some Linux distros. We decided to standardize on Mac and Windows to sidestep these issues. A more powerful MDM would enable us to remote wipe computers in case it is stolen, but we decided that the existing protections we had in place were sufficient to defend against this scenario.
+
+We came up with a few important policies. For example, Access Control Policy, Data Management Policy, Asset Management Policy, etc. Vanta has templates that we used as a starting point and adapted to our requirements. Since our auditors were familiar with the Vanta templates, there wasn’t much back and forth regarding the policy language.
```

Comment:
```suggestion
We came up with a few important policies. For example, Access Control Policy, Data Management Policy, Asset Management Policy, etc. Vanta has templates which we used as a starting point and adapted to our requirements. Since our auditors are familiar with the Vanta templates, there wasn’t much back and forth regarding the policy language.
```